tests/testthat/test-cobertura.R

```diff
@@ -3,6 +3,26 @@ context("cobertura_export")
 test_that("it works with coverage objects", {
   tmp <- tempfile()
   cov <- package_coverage(test_path("TestSummary"))
+
+  # fix host-dependent report parameters
```

Comment:
I would prefer not use use mocking when we can reasonably avoid it. After changing the use of `getwd()` as suggested in other comments to instead use the path attribute we could hard code the package path before the test here. e.g. `attr(cov, "package")$path <- "/dummy/directory"`

Then extend the readlines subsetting to exclude the first 3 lines rather than the first two, which would fix the other dynamic parts of the test.

Reply:
Done.